packages/documentation/pages/usage/components/comment.vue

```diff
@@ -79,7 +79,7 @@
 				</div>
 			</KtForm>
 
-			<pre><code>&lt;KtComment
+			<pre><code data-lang="vue">&lt;KtComment
```

Comment:
added data-lang wherever needed because it was otherwise dead and has long lived as a style